test/karma.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function(config) {
+  config.set({
+    frameworks: ['mocha', 'chai'],
+    files: [
+      '../dist/time-elements.js',
```

Comment:
We used to load the "legacy" file for tests, now we load the main one. I suspect this was because PhantomJS was unable to handle the non-legacy build.

However, if we load this test suite in an older browser, such as IE 11, would it work with this configuration?

Reply:
Good point. I don't think it would. Since the legacy file is the main file on the module, I have changed this here to test with the legacy build instead.